bin/MadGraph5_aMCatNLO/cards/production/13TeV/DarkMatter_Codex/Codex_LQ1000_DM_400_X_420_gen2/Codex_LQ1000_DM_400_X_420_gen2_proc_card.dat

```diff
@@ -34,6 +34,11 @@ define l+ = e+ mu+
 define l- = e- mu-
 define vl = ve vm vt
 define vl~ = ve~ vm~ vt~
+
+#import model sm-no_b_mass
```

Comment:
ciao, all the import commands are commented out, so you are not importing...
please fix